doc/changelog.txt

```diff
@@ -32,6 +32,63 @@ BREAKING CHANGES & HOW TO GET BACK OLD BEHAVIOURS:
      return originalGetConfigInt(key, def) or 0
    end
 
+ ! hovercraft and ships brought out of water no longer forced to be upright.
+   To get back previous behaviour, put the `upright = true` tag in all unit defs
+   whose move def is of the hovercraft or ship type.
+
+ ! The return value from the UnitUnitCollision callin is now ignored
+   and there is only one event for each collision.
+   There is no way to get back the old behaviour for now,
+   but if someone needs it it could be arranged.
+
+ ! The GameProgress callin is now called every 5 seconds (from 10).
+   This value is not exposed anywhere so to get back the previous
+   behaviour you will have to find whichever code assumes the old value.
+
+ ! Buildings can be placed inside other buildings' open yardmap gaps.
+   Use the AllowCommand or AllowUnitCreation callins to prevent this.
+
+ ! Rendering fonts now obeys GL color state. To get back previous
+   behaviour, add gl.Color in front of font calls.
+
+ ! removed the following constants:
+     Platform.glSupport16bitDepthBuffer
+     Platform.glSupport24bitDepthBuffer
+     Platform.glSupport32bitDepthBuffer
+   
+   To get back previous behaviour, replace with
+     Platform.glSupportDepthBufferBitDepth >= 16 -- or 24, or 32, respectively
+
+ ! removed LOD rendering (2D unit billboards when zoomed out far), including
+   the `/distdraw` command and the `UnitLodDist` springsettings entry
+
+ ! units with `useFootPrintCollisionVolume` but no `collisionVolumeScales` set
+   will now use the footprint volume (previously mistakenly used the model's sphere).
+   To keep the old hitvolume, set `useFootPrintCollisionVolume` to false for units
+   with no `collisionVolumeScales`. Assuming you apply `lowerkeys` to unit defs,
+   this can also be achieved by putting the following in unit defs post-processing:
+   
+   for unitDefID, unitDef in pairs(UnitDefs) do
+     if not unitDef.collisionvolumescales then
+       unitDef.usefootprintcollisionvolume = nil
+     end
+   end
+
+ ! removed the `AdvSky` springsetting and the `/dynamicsky` command.
```

Comment:
I'm not sure what this did, but the replication probably involves some cloud shader and/or skybox shenanigans so I wouldn't be able to list it anyway.

Reply:
With `/dynamicsky` clouds used to move on the sky, but the operation was slightly heavy on CPU.
`AdvSky` used to switch between BasicSky and AdvSky, both are gone.

Reply:
Alright, probably doesn't need a "how to get back" entry then.